src/apis/drive/package.json

```diff
@@ -4,16 +4,18 @@
   "description": "drive",
   "main": "build/index.js",
   "types": "build/index.d.ts",
-  "keywords": ["google"],
+  "keywords": [
+    "google"
+  ],
   "author": "Google LLC",
   "license": "Apache-2.0",
   "homepage": "https://github.com/google/google-api-nodejs-client",
   "bugs": {
-    "url" : "https://github.com/google/google-api-nodejs-client/issues"
```

Comment:
more a nit for the template, but this should be `googleapis` instead of `google` :)

Reply:
noted!